docs/manifest.json

```diff
@@ -96,6 +96,12 @@
 					"path": "./install/uninstall.md",
 					"icon_path": "./images/icons/trash.svg"
 				},
+				{
+					"title": "Install and test coder-preview",
+					"description": "Test the latest coder-preview build",
+					"path": "./install/coder-preview-docker.md",
+					"icon_path": "./images/icons/contributing.svg"
+				},
```

Comment:
How does this look in the docs sidebar? It's a pretty long title

Reply:
it's not great - and it might not be totally accurate. I don't think we actually install Coder with the `docker run` command

I'll update it in a little bit

Reply:
IMO, it should live as a notice of type "Tip" on the docker install page. And not a separate page.